tests/go.mod

```diff
@@ -195,8 +195,6 @@ replace (
 	cosmossdk.io/x/feegrant => ../x/feegrant
 	cosmossdk.io/x/nft => ../x/nft
 	cosmossdk.io/x/upgrade => ../x/upgrade
-	// This can be deleted after the CometBFT PR is merged
```

Comment:
ref: https://github.com/cosmos/cosmos-sdk/pull/15119, this was not removed would have surfaced e2e tests issue, that may or may not fixed with the current changes here.